docs/hil-bmi-commands.md

```diff
@@ -0,0 +1,41 @@
+# ESI Equivalency with HIL/BMI
+
+One of the goals of ESI is to replace [HIL/BML](https://github.com/CCI-MOC/hil). This document matches HIL/BMI commands with their OpenStack equivalents.
+
+* [Node Commands](#node)
+* [Network Commands](#network)
+
+## <a name="node"></a>Node Commands
+
+### Administrator
+
+* **hil node delete**: `openstack baremetal node delete`
+* **hil node nic delete/register**: ?? `openstack baremetal port create/delete`
+* **hil node register**: `openstack baremetal node create`
+
+### User
+
+* **hil node bootdev**: ??
+* **hil node console**: `openstack baremetal node console show`
+* **hil node list**: `openstack baremetal node list`
+    * showing free nodes may require ESI leasing service
+* **hil node metadata add/delete**: ??
+* **hil node network connect/detach**: will be replaced by an ESI OpenStack equivalent
+* **hil node obm disable/enable**: ??
```

Comment:
no need of an equivalent I think. This command just enables the obm service for a particular HIL node. We don't need to explicitly manage that in my opinion.

Reply:
Okay, great!